tests/src/test_cases/test_txt.c

```diff
@@ -0,0 +1,21 @@
+#if LV_BUILD_TEST
+#include "../lvgl.h"
+
+#include "unity/unity.h"
+
+void test_txt_should_identify_empty_string_txt_parameter_when_trying_to_get_next_line(void);
+
+/* See #2615 for more information */
+void test_txt_should_identify_empty_string_txt_parameter_when_trying_to_get_next_line(void)
```

Comment:
It seems a bit long to me... how about this?

```suggestion
void test_txt_next_line_handles_empty_string(void)
```

Reply:
That's much better, thanks for the suggestion.